### PR TITLE
Revert "the team controller was stuck on a very old version of jx."

### PR DESCRIPTION
**Pull request by jtnord**
Reverts jenkins-x/jenkins-x-platform#4398

### CODE REVIEW

**Comment by jtnord** [comment]
the `jx` image does not have `helm` installed which is needed for the team controller.

```
error: failed to initialise helm: failed to run 'helm init --client-only' command in directory '', output: '': exec: "helm": executable file not found in $PATH
```

File: values.yaml [hunk 1]
```diff
@@ -409,6 +409,9 @@ controllerteam:
   args:
   - "controller"
   - "team"
+  image:
+    repository: jenkinsxio/builder-go
+    tag: 0.0.498
```
Comment (jstrachan) [inline values.yaml]: may as well switch to a more recent image while we're at it? e.g. `0.1.113` 
https://hub.docker.com/r/jenkinsxio/builder-go/tags/

Comment (jtnord) [reply values.yaml]: this is a pure revert from GH.
I don't really want to mix a revert commit with changes.
will do a follow up for that.

**Pull request merged**